{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/settings.py

```diff
@@ -293,7 +293,7 @@
     AWS_STORAGE_BUCKET_NAME = config("AWS_STORAGE_BUCKET_NAME")
     AWS_SECRET_ACCESS_KEY = config("AWS_SECRET_ACCESS_KEY")
     AWS_S3_CUSTOM_DOMAIN = AWS_STORAGE_BUCKET_NAME + ".s3.amazonaws.com"
-    AWS_LOCATION = config("AWS_LOCATION", default="")
+    AWS_LOCATION = config("AWS_LOCATION", default="")  # production, staging, etc
```

Comment:
@oudeismetis actually can we let this error? We had a problem in Lighten where I did not set it and so we kept getting rollbar errors that were indicating the error was with a trailing slash but really it was because Location was missing!

Reply:
yea we can switch to that. This is a pretty critical setting which is why I added the code comment.